action.yml

```diff
@@ -86,6 +91,7 @@ runs:
     - --excludes=${{ inputs.excludes }}
     - --github-token=${{ inputs.githubtoken }}
     - --html-report-file=${{ inputs.patchdir }}/report.html
+    - --ignore-unknown-options=true
```

Comment:
Are `ignore-` and `report-unknown-options` mutually exclusive? Can test-patch infer one from the other?

Reply:
They are not. `--ignore-unknown-options` allows you to pass options that test-patch may not know about.  `--report-unknown-options` is whether or not test-patch tells you that unknown options got passed.  I debated whether or not `--report` should auto trigger `--ignore` but felt for now to make `--report` working dependent upon `--ignore`.  This is relatively easy to change later if we find out it doesn't work very well.